roles/install_ca/tasks/main.yml

```diff
@@ -22,6 +22,12 @@
         path: "{{ cifmw_install_ca_trust_dir }}"
         state: directory
 
+    - name: Install internal CA from url
+      when: cifmw_install_ca_url is defined
+      ansible.builtin.get_url:
```

Comment:
we may want to allow to not check TLS certificate at this point?

Reply:
I'm not sure what you're asking, is there some parameter to `get_url` that it's missing?

Reply:
I missed that parameter, thanks @cjeanner updated the patch with a variable that controls whether we validate the certs